src/messages.js

```diff
@@ -61,7 +61,11 @@ const messages = {
   insufficientFunds: 'You have insufficient funds to perform transaction 💸\n Top up with small amount and retry',
   genericError: (message) => `An error occurred: ${message}`,
   selectActionQuestion: 'You can now perform one of the following actions',
+  changeUrlInfo: (oldUrl, newUrl, warning) => `You will now change your node URL from ${oldUrl} to ${newUrl}.\n${warning}`,
+  changeUrlWarning: '⚠️ WARNING!⚠️ This operation will cost you gas. Please, check again if provided URL is valid.',
```

Comment:
Gas? You mean nectar? Btw can we really assume, that everyone knows what gas/nectar is? Maybe we should rather estimate (more or less) operation cost and print sth like `This operation will cost you ~0.01 AMB, make sure your account is not empty`.

Reply:
Also I'd split this warning. Now it looks like these two warnings are somehow connected.
```
changeUrlWarning: '⚠️ WARNING!⚠️ This operation will cost you <WHATEVER_HERE>.\nAdditionally, please, double-check if provided URL is valid.',
```
Or even split it into `gasCostWarning` and `changeUrlWarning `

Reply:
👍